server/devdocs/Makefile

```diff
@@ -1,12 +0,0 @@
-NODE_BIN := $(shell npm bin)
-MOCHA ?= $(NODE_BIN)/mocha
-BASE_DIR := $(NODE_BIN)/../..
-NODE_PATH := test:$(BASE_DIR)/client
-COMPILERS ?= js:babel/register
-REPORTER ?= spec
-UI ?= bdd
-
-test:
-	@NODE_ENV=test NODE_PATH=$(NODE_PATH) $(MOCHA) --compilers $(COMPILERS) --reporter $(REPORTER) --ui $(UI)
-
-.PHONY: test
```

Comment:
it was left over from another PR where it _should_ have been deleted. there are no tests left so the Makefile should go too